src/main/org/apache/tools/ant/taskdefs/compilers/JavacExternal.java

```diff
@@ -94,9 +94,8 @@ private int moveArgFileEligibleOptionsToEnd(String[] commandLine) {
         for(int i = nonArgFileOptionIdx + 1; i < commandLine.length; i++) {
             if (!isArgFileEligible(commandLine[i])) {
                 String option = commandLine[i];
-                for(int j = i - 1; j >= nonArgFileOptionIdx; j--) {
-                    commandLine[j + 1] = commandLine[j];
-                }
+                if (i - nonArgFileOptionIdx >= 0)
```

Comment:
please add braces even for single line blocks

Reply:
HI @bodewig 
I just add the braces.
TY